modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/persistence/impl/ListTypeDefinitionPersistenceImpl.java

```diff
@@ -2630,8 +2630,7 @@ public ListTypeDefinition updateImpl(
 			if (isNew) {
 				if (ercListTypeDefinition != null) {
 					throw new DuplicateListTypeDefinitionExternalReferenceCodeException(
-						"Duplicate list type definition with external reference code " +
-							listTypeDefinition.getExternalReferenceCode());
+						"This External Reference Code is already in use. Try another one.");
```

Comment:
Hi @daiazius there is some reason to change this message?

Reply:
I changed it to be the same as requested in the issue